### ISSUE TITLE
Convert BOOL, TRUE, FALSE, etc.

### ISSUE THREAD

**Issue by maxolasersquad** [linappleii/linapple#68]
In some files, e.g. `Registry.cpp` there are uses of VOID, BOOL, etc. Some Googling indicates these are artifacts of win32 programming. What I couldn't tell for sure is if it is safe to just move these to the standard lower-case versions. It looks like VOID can safely be converted, but I'm not so sure about the other keywords.

**Comment by ghedger** [comment]
As a general rule, I'm in favor of anything overtly Windows-ey becoming closer to language intrinsics (e.g. VOID becomes void, BOOL becomes bool, FALSE -> false, etc.) It's probably best to do these kinds of wide-ranging changes now or in the very near future, not too far removed from the wholesale formatting changes you completed.

**Comment by rhaleblian** [comment]
Now might be an OK time? Now that Travis is here he can help. :P

**Comment by maxolasersquad** [comment]
I have addressed a lot of the win32 types that originate from the `wincompat.h` file in https://github.com/linappleii/linapple/pull/123 This doesn't cover everything, but I believe it covers all of the major types. Types such as `typedef WORD far *LPWORD` I'm not sure how to convert properly so I just left them alone.

**Comment by rhaleblian** [comment]
Nice.

**Comment by maxolasersquad** [comment]
If anyone can instruct me on how to convert the remaining Win32 types I'd be happy to do the grunt work. My dev background is primarily in interpreted languages like Python and Javascript so this kind of stuff isn't exactly in my wheel-house.

**Comment by rhaleblian** [comment]
Lemme look for WIN32-isms up in here...

**Comment by rhaleblian** [comment]
Sorry no time.
I would say put this in? It's a step on the journey.

**Comment by akhepcat** [comment]
> There are 13 remaining uses of `BOOL`, all in `/src/CPU.cpp`. There are no remaining usages of `TRUE` or `FALSE`.

most of those are for the emulated CPU, and aren't traditional booleans.   it'll be fun to break those down and see what can be converted without breaking the CPU...

I guess I can take a look at it?

**Comment by akhepcat** [comment]
Yeah, i stole the easy ones first, lol.   the flags are gonna be ...  fun.

**Comment by maxolasersquad** [comment]
This is now done.
